lib/trebuchet/backend/memcached.rb

```diff
@@ -13,11 +13,11 @@ def get_strategy(feature_name)
     @memcache.get(key(feature_name))
   end
 
-  def set_strategy(feature, strategy, options = nil)
+  def set_strategy(feature, strategy, options = nil, force = false)
```

Comment:
:x: don't use default positional arguments for this.  https://github.com/airbnb/ruby#no-default-args